lib/graphql/mutations.js

```diff
@@ -108,37 +109,21 @@ export const deleteApplicationMutation = gql`
   }
 `;
 
-export const addCreateCollectiveMutation = graphql(createCollectiveQuery, {
+export const addCreateCollectiveMutation = gql(createCollectiveQuery, {
```

Comment:
To simplify migration to the new flow, this should be left as it was.

The modified version looks god but should be either available as `addCreateCollectiveMutationV2` or directly copy/pasted where it's used.